iis-wf/iis-wf-documentsclassification/src/main/java/eu/dnetlib/iis/wf/documentsclassification/DocumentClassificationJob.java

```diff
@@ -39,73 +37,59 @@
     private static DocumentToDocClassificationMetadataConverter converter = new DocumentToDocClassificationMetadataConverter();
     
     private static DocClassificationReportGenerator reportGenerator = new DocClassificationReportGenerator();
-    
-    
+
     //------------------------ LOGIC --------------------------
     
     public static void main(String[] args) throws IOException {
-        
         DocumentClassificationJobParameters params = new DocumentClassificationJobParameters();
         JCommander jcommander = new JCommander(params);
         jcommander.parse(args);
-        
-        
-        SparkConf conf = new SparkConf();
-        conf.set("spark.serializer", "org.apache.spark.serializer.KryoSerializer");
-        conf.set("spark.kryo.registrator", "pl.edu.icm.sparkutils.avro.AvroCompatibleKryoRegistrator");
-        
-        
-        try (JavaSparkContext sc = new JavaSparkContext(conf)) {
-            
+
+        try (JavaSparkContext sc = JavaSparkContextFactory.withConfAndKryo(new SparkConf())) {
             HdfsUtils.remove(sc.hadoopConfiguration(), params.outputAvroPath);
             HdfsUtils.remove(sc.hadoopConfiguration(), params.outputReportPath);
+
+            sc.sc().addFile(params.scriptDirPath, true);
```

Comment:
I've branched from master with `if` condition, so with #984 issue fix, but to my great surprise and after spending 3 hours I've found out that with spark 2 the fix has to removed and explicit script dir path has to be reintroduced to make all the tests in this module pass again. Also with explicit script dir path primary workflow integration tests pass. However, I did not explicitly run primary workflow, so I think this could be done as an ultimate test.

Reply:
OK, let's mention about this explicitly in task message referencing the #984 issue so we would know it was reverted.